### DISCUSSION TITLE
Chia Client hangs spinning on Full Node TAB forever

### DISCUSSION THREAD

**Discussion by Murf** [Chia-Network/chia-blockchain#5043] [Support]
Hi All,
Under FullNode Status, all three items (Status, Blocks and Connections) stay a spinning circle forever.
I am running 1.1.5 of the Windows client
I have tried the following:

* Remove C:\Users\myusername\ .chia
* Remove chia from Credentials manager
* Re-installed
* Checked port 8444 is open, it is

I am getting the following error in the client logs:
2021-05-15T11:09:42.724 wallet wallet_server              : INFO     Cannot connect to host 127.0.0.1:8444 ssl:True [SSLCertVerificationError: (1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate (_ssl.c:1091)')]

Thing to note: I am using Swar-Chia-Plot-Manager to do the plotting in the background, I specifically set my keys manually in that so it all works fine.

Any suggestions most welcome.
Murf.

**Comment by Murf** [comment]
Why have a support if no one anwsers?

**Reply by tiny0little** [reply]
what support?

**Comment by Murf** [comment]
The category I picked?

**Reply by tiny0little** [reply]
I don't think there are dedicated CHIA stuff who is sitting and watching this discussions 8x5
you are getting answers from your fellow farmers who might be half a step ahead

**Reply by Murf** [reply]
Misnamed category, should be "Community Help"

**Comment by BrandtH22** [comment]
@Murf , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Note - that error generally indicates the database is corrupt. I would remove the current database and either resync from genesis or use the official torrent https://www.chia.net/downloads/#database-checkpoint

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
